includes/extra_datafiles/dist.site-specific-bootstrap-settings.php

```diff
@@ -23,3 +23,8 @@
 // (true) to disable the back-to-top control on each page.
 //
 //$zca_disable_back_to_top = false;
+
+// ----
+// This controls preloading of fontawesome, jquery and boostrap css
+// if you don't want to preload the scrips and css uncomment line below
+//$bs4_no_preloading = true;
```

Comment:
Let's keep with the current naming convention for variables here, use `$zca_no_preloading` and (like the other soft-settings) the value present here is the default (`false`).